newrelic-agent/src/main/resources/newrelic.yml

```diff
@@ -284,19 +285,15 @@ common: &default_settings
       # A comma separated list of attribute keys whose values should not be sent to New Relic.
       #exclude:
 
-  # Cross Application Tracing adds request and response headers to
+  # Cross-application tracing adds request and response headers to
```

Comment:
Other docs pages seem to be using a non-hyphenated convention.  Which is preferred?
https://docs.newrelic.com/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file/#Cross_Application_Tracer

Reply:
I'm fine following that pattern. In general, in English, we hyphenate multi-word adjectives :) I'm removing the hyphen.

Reply:
Cross Application Tracing is a proper name for the feature and the hyphenation rule doesn't apply to compound adjective statements of proper nouns. But, in any case, my main point is that it should be consistent throughout our docs and in the yml. My main concern is consistency.